packages/ethereum-contracts/package.json

```diff
@@ -29,16 +29,16 @@
     "typings": "./types/index.d.ts",
     "scripts": {
         "dev": "tasks/dev.sh",
+        "clean": "rm -rf node_modules; rm -rf cache; rm -rf build; rm -rf artifacts; rm -rf typechain-types; rm -rf scripts/*.d.ts scripts/*.d.ts.map",
         "run-hardhat": "IS_HARDHAT=true hardhat",
         "run-truffle": "IS_TRUFFLE=true truffle",
         "run-forge": "IS_FOUNDRY=true forge",
         "run-nodemon": "nodemon -e sol,js,ts -x",
         "build": "run-s build:*",
-        "build:contracts": "rm -f build/contracts/*;yarn build:contracts-truffle",
         "build:contracts-truffle": "yarn run-truffle compile",
         "build:contracts-hardhat": "yarn run-hardhat compile",
         "build:typechain": "tsc -p tsconfig.typechain.json",
-        "build:scripts": "tsc -p tsconfig.scripts.json",
+        "build:scripts": "rm -rf scripts/*.d.ts scripts/*.d.ts.map; tsc -p tsconfig.scripts.json",
```

Comment:
my only concern with this is that it will break when trying to remove the .d.ts file and .d.ts.map files if they don't exist, I have a solution I worked on locally, but removed, I will add it tomorrow, but it involves writing a bash script which checks existence of the .d.ts in scripts and does the remove based on that